addons/spike/XEH_postInit.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+
+if (hasInterface) then { 
+    #include "initKeybinds.sqf"
+
+    // add camera interactions
+    private _switchToCameraAction = ["SwitchToCamera", "Switch To Missile Camera", "", {
+        // statement
+        params ["_target", "_player", "_params"];
+        private _camera = _player getVariable [QGVAR(missileCamera), objNull];
+        [_camera] call FUNC(camera_switchTo);
+    }, {
+        // condition
+        params ["_target", "_player", "_params"];
+        private _camera = _player getVariable [QGVAR(missileCamera), objNull];
+        private _projectile = _camera getVariable [QGVAR(missile), objNull];
+        !([] call FUNC(camera_userInCamera)) && { !(_camera isEqualTo objNull); } && { !(_projectile isEqualTo objNull) }
```

Comment:
All `!(... isEqualTo ...)` instances could be replaced by `... isNotEqualTo ...`.

Reply:
Checks against `objNull` can just be `isNull`